front/src/styles/scss/applications/stdcm/_results.scss

```diff
@@ -312,6 +312,18 @@
       display: flex;
       flex-direction: column;
       min-height: 10.5rem;
+      width: 70%;
+      color: var(--grey80);
+
+      ul {
+        list-style: unset;
+        margin-left: 1.25rem;
+
+        li span {
+          position: relative;
+          left: -0.25rem;
+        }
+      }
```

Comment:
We can simply add `&bull;` in `span` because the size of the list point is larger than that of the model and the alignment is not correct.

```suggestion
```